packages/pwa-devdocs/src/technologies/basic-concepts/app-shell/index.md

```diff
@@ -1,12 +1,17 @@
 ---
 title: Application Shell
+contributors:
+    - gavin2point0
 ---
 
-The Magento PWA Studio uses application shell architecture to quickly load pages on mobile apps.
+Magento PWA Studio uses an application shell architecture to shorten the time it takes to load pages.
 
-This Approach involves heavily caching the minimal amount of HTML, CSS and JS to load the basic UI of the page. Then pulling in the rest of the content through an API. 
+This approach involves heavily caching the minimal amount of HTML, CSS and JS to load the basic UI of the page before fetching the rest through an API. 
 
-Because the majority of the page is cached, it will load almost instantly on repeat visits. It also prevents unnecessary data usage as it removes the need to download any static content more than once.
+Page load is instantaneous on repeat visits because the majority of the page is in the cache.
```

Comment:
Similar to my previous comment:

> Page load is instantaneous

Might be less ambiguous to say "rendering of the application shell is instantaneous" or something to that effect